jsonnet/utils/sanitize-rules.libsonnet

```diff
@@ -183,6 +183,27 @@ local patchedRules = [
         },
         'for': '30m',
       },
+      {
```

Comment:
Could you add a comment explaining why we override the alert and a link to the pod installer bug (not sure if we have it already)?